build/system.gyp

```diff
@@ -11,6 +11,19 @@
   },
   'targets' : [
     {
+      'target_name': 'linux_notification',
+      'type': 'none',
+      'conditions': [
+        ['tizen!=1', {
```

Comment:
There's already a check for tizen==1 after the "gio" target. I suppose you can just add it to the "else" part of it instead of adding yet another check inside the target.